### DISCUSSION TITLE
How to use the xaml file in the language directory?

### DISCUSSION THREAD

**Discussion by liuzhichao03** [nomi-san/parsec-vdd#65] [Q&A] [answered]
How to use the xaml file in the language directory?

**Comment by nomi-san** [comment]
What will you do with it?

If you need to add a new language, clone `en.xaml` to new file `[id].xaml`, edit your translation and rebuild the app. It will appear in the language selection.

**Reply by liuzhichao03** [reply]
How to use the zh.xaml Chinese language file on an existing program?

**Reply by nomi-san** [reply, marked as answer]
You need to clone the latest source code and build it manually.
However, you can download it from the latest build action.
https://github.com/nomi-san/parsec-vdd/actions/runs/10615568256

![image](https://github.com/user-attachments/assets/ab2091e2-bb2e-43f2-ac38-13490fbab4af)
